reference/configuration-files.rst

```diff
@@ -231,6 +231,31 @@ After a Git push, you can see the results of the deployment hooks in the ``/var/
 
 ----
 
+A good example combining dependencies and hooks: a theme compiling sass files using grunt. Here is an example of how to set this up:
```

Comment:
As a good example combining dependencies and hooks, you can compile your SASS files using Grunt.